scripts/manifests/thunderbirdmanifest.json

```diff
@@ -47,7 +47,7 @@
   "applications": {
     "gecko": {
       "id": "unicodify@rugk.github.io",
-      "strict_min_version": "87.0"
+      "strict_min_version": "91.0"
```

Comment:
why the version bump?

Reply:
Thunderbird 87 was a beta only version. Thunderbird releases follow the Firefox ESR versions, so 78, 91, 102, etc...